config/settings.d/puppet.yml.example

```diff
@@ -41,16 +41,20 @@
 # If you want to override the puppet_user above just for mco commands
 #:mcollective_user: peadmin
 
-# URL of the puppet master itself for API requests
+# Override use of Puppet's API to list environments, by default it will use only if
+# environmentpath is given in puppet.conf, else will look for environments in puppet.conf
+# (Puppet versions prior to 4.0 only)
+#:puppet_use_environment_api: true
+
+# Puppet master version used
```

Comment:
Yeah, I wasn't sure what to use, I figured most people would know what version of puppet they are running straight away, and we know what version of puppet server (or more importantly version of api) that corresponds too. Will drop 'master' from the comment.